recipes/x86_64_level/conda_build_config.yaml

```diff
@@ -0,0 +1,59 @@
+microarchitecture:
+- x86_64
+- x86_64_v2
+- x86_64_v3
+- x86_64_v4
+- nocona
+- core2
+- nehalem
+- westmere
+- sandybridge
+- ivybridge
+- haswell
+- broadwell
+- skylake
+- mic_knl
+- skylake_avx512
+- cannonlake
+- cascadelake
+- icelake
+- k10
+- bulldozer
+- piledriver
+- steamroller
+- excavator
+- zen
+- zen2
+- zen3
+
+level:
+- 1
+- 1
+- 2
+- 3
+- 1
+- 1
+- 2
+- 2
+- 2
+- 2
+- 3
+- 3
+- 3
+- 3
+- 4
+- 3
+- 4
+- 4
+- 1
+- 2
+- 2
+- 2
+- 3
+- 3
+- 3
+- 3
```

Comment:
Can we add some validation for this mapping, or at least order all the non-x86_64 architectures by level? E.g.
```
# the x86_64 feature levels that everything is mapped to, see
# https://en.wikipedia.org/wiki/X86-64#Microarchitecture_levels
- x86_64
- x86_64_v2
- x86_64_v3
- x86_64_v4
# architectures (as recognised by archspec) corresponding to level 1
- core2
- nocona
- ...
# level 2
- ...
# level 3
- ...
# level 4
- ...

Reply:
Or perhaps we could avoid the zip completely by using `core2:1` / `nocona:1` etc., and then splitting on `:`

Reply:
It's auto-generated by the script given. Could you say what sort of validation you had in mind? (other than ordering)

Reply:
I didn't notice right away that this was generated - that's more than enough in terms of validation.

I just prefer leaving more breadcrumbs for other contributors (including our future selves), so perhaps it'd be helpful to include a comment on top of the cbc that this shouldn't be edited manually, but needs to be generated using `gen.py`